internal/handlers/sqlite/msg_createindexes.go

```diff
@@ -513,7 +519,8 @@ func validateIndexesForCreation(command string, existing, toCreate []backends.In
 			existingKey := formatIndexKey(existingIdx.Key)
 
 			if newIdx.Name == existingIdx.Name && newKey == existingKey {
-				// Fully identical indexes are allowed.
+				// Fully identical indexes are ignored, no need to attempt to create them.
+				toCreate = slices.Delete(toCreate, i, i+1)
```

Comment:
We are iterating over `toCreate` in the outer loop. Wouldn't deleting element `i` from the `toCreate` slice cause next iteration to skip new `i` element, because next iteration would check `i+1`? :thinking:

Reply:
yeah, you are right! nice catch!

Reply:
Added a test that would demonstrate the problem (see `DuplicateAndInvalid`) and added a fix.